.travis.yml

```diff
@@ -16,6 +16,10 @@ script:
   - cp config/example.settings.yml config/settings.dev.yml
   - php vendor/bin/phpunit --group UnitTest
   - php vendor/bin/phpcs --standard=PSR2 src test
+  - php vendor/bin/phpunit --coverage-clover=coverage.xml
```

Comment:
You can combine this with the above phpunit line so it only runs once
`php vendor/bin/phpunit --group UnitTest --coverage-clover=coverage.xml`